mne_bids/utils.py

```diff
@@ -591,12 +591,14 @@ def _find_matching_sidecar(bids_fname, bids_root, suffix, allow_fail=False):
 
     # We only use subject and session as identifier, because all other
     # parameters are potentially not binding for metadata sidecar files
-    search_str = 'sub-' + params['sub']
+    sub_id = 'sub-' + params['sub']
+    search_str = sub_id
```

Comment:
I don't understand this change. Why do you have two variables that point to the same thing?